include/oneapi/dpl/algorithm

```diff
@@ -13,8 +13,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _ONEDPL_ALGORITHM
```

Comment:
I have a concern that variable can be used in the some code for some workaround somewhere....
(for example To check OnDPL was included or not..)

Reply:
I this that your concern is too strict: I am tried too find all of changed defined at all gitHub and they was found only in our code.